caput/memh5.py

```diff
@@ -2239,6 +2239,13 @@ def deep_group_copy(
             else:
                 data = entry[selection]
 
+            # only apply if conversion is h5py.Group -> memh5.Group
+            # reading from h5py can result in arrays with explicit endian set
+            # which mpi4py cannot handle when Bcasting memh5.Group
+            # needed until fixed: https://github.com/mpi4py/mpi4py/issues/177
+            if isinstance(g1, h5py.group) and isinstance(g2, MemGroup):
```

Comment:
There's obviously a typo here (lower case g), which is presumably why the tests fail.

However, to prepare for a zarr world I wonder if we should change the first clause to `not instance(g1, MemGroup)`.

Also, I'm not sure what zarr does in terms of typing of arrays, so maybe we don't need to do it.

Reply:
Looks like `zarr` gives an array with native order, so we don't need to do anything at all. In which case your version (with the fixed typo) is all good.